src/gui/owncloudgui.cpp

```diff
@@ -713,5 +712,17 @@ void ownCloudGui::slotRemoveDestroyedShareDialogs()
     }
 }
 
+void ownCloudGui::updateDialogVisibility(QDialog *dialog, bool visible)
+{
+    if (dialog) {
+        auto index = _visibleDialogs.indexOf(dialog);
```

Comment:
Should be `const auto` here. Also for readability I'd have a `const auto found = index >= 0;` or similar and use that in the ifs.

Reply:
done